packages/helm-charts/testnet/templates/validators.statefulset.yaml

```diff
@@ -94,6 +94,11 @@ spec:
           echo -n "Generating Bootnode enode for the validator: "
           cat /root/.celo/bootnodeEnode
 
+          # Generate a fee recipient address if available
+          if celotooljs.sh generate address-from-env --help | grep -q tx_fee_recipient; then
```

Comment:
Why is this if necessary?  I thought if we're hitting this code it means we're on this branch (or it's been merged already), and so the if would always come out true.  Am I missing something?

Reply:
Not necessarily. Celo-monorepo container version is defined in the .env file, and it is possible and I'd say frequent to deploy from a commit that can contain a new update of the charts, but having an old version of the monorepo commit in the .env file. It is also possible to have otherwise.